plugins/txn_test_gen_plugin/txn_test_gen_plugin.cpp

```diff
@@ -120,9 +120,9 @@ struct txn_test_gen_plugin_impl {
 
          trx.actions.emplace_back(vector<chain::permission_level>{{creator,"active"}}, contracts::newaccount{creator, newaccountC, owner_auth, active_auth, recovery_auth});
          }
-      
+
          trx.expiration = cc.head_block_time() + fc::seconds(30);
-         trx.net_usage_words = 500;
+         trx.max_net_usage_words = 500;
```

Comment:
I think we should just 0 these out, the txn_test_gen_plugin isn't concerned with abusive contracts.